commercetools-models/src/main/java/io/sphere/sdk/products/search/PriceSelectionBuilder.java

```diff
@@ -80,6 +86,16 @@ public PriceSelectionBuilder priceChannel(@Nullable final Referenceable<Channel>
         return this;
     }
 
+    public PriceSelectionBuilder storeProjection(@Nullable final String storeProjection) {
+        this.storeProjection = storeProjection;
+        return this;
+    }
+
+    public PriceSelectionBuilder localeProjection(@Nullable final String localeProjection) {
+        this.localeProjection = localeProjection;
```

Comment:
Here you would create a new List for the `localeProjection` and add the given `localeProjection` to it.

Reply:
Here we should add another method :
```java
    public PriceSelectionBuilder localeProjection(@Nullable final String localeProjection) {
        this.localeProjection = new ArrayList<>();
        this.localeProjection.add(localeProjection);
       return this;
   }
```
This would align with our existing code and makes it easier to start with a single value.

Reply:
ok, I'm gonna update my method `plusLocaleProjection` in this way

Reply:
I just saw the usage also in other builders as they are immutable too.